docs/release-notes/release-notes-1.0.1.md

```diff
@@ -0,0 +1,41 @@
+# DCRDEX v1.0.1
+
+This release contains several important fixes and improvements.  All users are advised to update.
+
+Please read the [v1.0.1 release notes](https://github.com/decred/dcrdex/releases/tag/v1.0.0) for a full list of changes since v0.6.
+
+## Fixes and Improvements
+
+- Allow collapsing user-expanded reputation info on /markets
+- Removed unused changepos field in btc fundrawtransaction that was causing a JSON parse error when firod returned a negative value
+- Update minimum Firo version to 0.14.14.0
+- Add support for enabling/disabling a server
+- Fix a couple of issues with the Binance market feed subscriptions
+- Fix a lock-contention problem where a long running "tick" function was causing the client to fail to report their preimage
```

Comment:
I thought we only fixed the possibility of panics on cancel orders when the tick was too long. Did you fix the long tick problem? Maybe I missed that.

Reply:
We didn't fix the long ticks themselves. We made it so that long ticks don't inhibit the client from responding to preimage requests when the `trackedTrade.mtx` was blocking access to the preimage for cancel orders.

Reply:
Ok I see now. I was getting the trade and tracker mutexes mixed up again.